code/modules/paperwork/photocopier.dm

```diff
@@ -203,6 +221,40 @@
 		visible_message("<span class='notice'>A red light on \the [src] flashes, indicating that it is out of toner.</span>")
 	return p
 
+
+/obj/machinery/photocopier/proc/copyass()
+	var/icon/temp_img
+	if(ishuman(ass) && (ass.get_item_by_slot(slot_w_uniform) || ass.get_item_by_slot(slot_wear_suit)))
+		usr << "<span class='notice'>You feel kind of silly copying [ass == usr ? "your" : ass][ass == usr ? "" : "\'s"] ass with [ass == usr ? "your" : "their"] clothes on.</span>"
+		return
+	if(check_ass()) //You have to be sitting on the copier and either be a xeno or a human without clothes on.
+		if(isalien(ass) || istype(ass,/mob/living/simple_animal/hostile/alien)) //Xenos have their own asses, thanks to Pybro.
+			temp_img = icon("icons/ass/assalien.png")
+		else if(ishuman(ass)) //Suit checks are in check_ass
+			if(ass.gender == MALE)
+				temp_img = icon("icons/ass/assmale.png")
```

Comment:
no that's not the thing I know they exist
but these are using icon + double quotes
files don't get compiled into the rsc in double quotes, which means this will only work if the icons/ass folder is present on the server
(which obviously it is from how our github works, but still, it's odd)